### DISCUSSION TITLE
Please add to Fooocus

### DISCUSSION THREAD

**Discussion by Richt12** [lllyasviel/Fooocus#850] [Ideas]
Fooocus Needs a face swapper and a metadata prompt saver

1.Face swapper to swap faces with a generic character(s)

2.Save the prompt you typed to the file to recall later use if you decide to go back to it later

**Comment by AkoZ** [comment]
* for prompt there is a log.html file .. which is right good inside the dir you put your images.. 
it's a bit different than to have inside .png but Ill' put some ethical arguments to not make this here ...
